app/models/concerns/orchestration.rb

```diff
@@ -82,14 +90,26 @@ def record_conflicts
     @record_conflicts ||= []
   end
 
+  def skip_orchestration!
+    @skip_orchestration = true
+  end
+
+  def enable_orchestration!
+    @skip_orchestration = false
+  end
+
+  def skip_orchestration?
+    @skip_orchestration ||= Rails.env == "test"
```

Comment:
If `@skip_orchestration` is `false` (i.e. a test calls `enable_orchestration!`) then this will return the value of `Rails.env == "test"` over `@skip_orchestration`, which I don't think is the intention (the value of `@skip_orchestration` should be returned).

The code should check if `@skip_orchestration.nil?` and only then return `Rails.env == 'test'`, else return `@skip_orchestration`.